### PR TITLE
Add management command to sync users data wrt to edly requirements

### PR DESCRIPTION
**Pull request by danialmalik**
**Description:** 
Add a management command to create appropriate roles and organization for existing users.

**JIRA:** 
https://edlyio.atlassian.net/browse/EDE-749

### CODE REVIEW

File: openedx/features/redhouse_features/management/commands/sync_users_data.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                'Created CourseCreator for user %s with status %s',
+                user.username,
+                CourseCreator.GRANTED
+            )
+
+            return course_creator
+
+    def create_course_access_roles(self, user, org):
+        try:
+            course_access_role = CourseAccessRole.objects.get(user=user)
```
Comment (HamzaIbnFarooq) [inline openedx/features/redhouse_features/management/commands/sync_users_data.py:121]: We want to check specific role, so this query will not work here at all.

File: openedx/features/redhouse_features/management/commands/sync_users_data.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            log.info(
+                'Created edly user profile for user %s and linked with sub org %s',
+                user.username,
+                sub_org
+            )
+            return edly_profile
+
+    def create_course_creator(self, user):
+        try:
+            course_creator = CourseCreator.objects.get(user=user)
```
Comment (HamzaIbnFarooq) [inline openedx/features/redhouse_features/management/commands/sync_users_data.py:97]: What if the user is present in CourseCreator but hasn't granted the access yet?

File: openedx/features/redhouse_features/management/commands/sync_users_data.py (hunk 1 above)
Comment (danialmalik) [reply openedx/features/redhouse_features/management/commands/sync_users_data.py:121]: This is just a first time fix for the existing users. This command is not intended to be used in future therefore its very unlikely that there is an existing access role for a user.
And even if there is (in this case), we don't want to override/change that existing one.

File: openedx/features/redhouse_features/management/commands/sync_users_data.py (hunk 2 above)
Comment (danialmalik) [reply openedx/features/redhouse_features/management/commands/sync_users_data.py:97]: As mentioned above, its very unlikey, but if this is the case, that object will be handled manually.

File: openedx/features/redhouse_features/management/commands/sync_users_data.py (hunk 1 above)
Comment (HamzaIbnFarooq) [reply openedx/features/redhouse_features/management/commands/sync_users_data.py:121]: I didn't get it, however leaving it onto you.

**Pull request merged**